### ISSUE TITLE
Unable to sync testnet:  Merkle check failed

### ISSUE THREAD

**Issue by bitfag** [GolosChain/golos#278]
I dunno is this a bug in daemon or not, but it definetely should not happen:

```
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17878 (id:000045d611e75c7170c3b3c756f05334b7396e
f6)
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17879 (id:000045d7a9f75dfefe9baa724c56acee29ff7a
40)
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17880 (id:000045d8ba200a2b07ec0a4d99a820e09e5319
d7)
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17881 (id:000045d9c03d267ec433353a4b752972d72bb0
51)
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17882 (id:000045da207ba8a1e8676e546f69b4cf4ae1e5
b5)
1954645ms p2p        node.cpp:3117                 send_sync_block_to_n ] Successfully pushed sync block 17883 (id:000045dbd3c28fe349c6a6353f05d834da2e32
dc)
1954656ms th_a       database.cpp:2860             _apply_block         ] 10 basic_exception: tion
next_block.transaction_merkle_root == merkle_root: Merkle check failed
    {"next_block.transaction_merkle_root":"f2d8df34360e45414f15b97c70b73179ec2f2905","calc":"4e984e8dba1af379812ce7c47328beb5b1f375a5","next_block":{"previous":"000045dbd3c28fe349c6a6353f05d834da2e32dc","timestamp":"2017-11-15T11:48:30","witness":"cyberfounder","transaction_merkle_root":"f2d8df34360e45414f15b97c70b73179ec2f2905","extensions":[],"witness_signature":"20523519bb5462ece023573933f988b9023f699765e1ca8597a511ace002526ffc4afdc60d12199294de43178fffe7af9d75e3ffa3f88b3296ad0f05eae1d009c5","transactions":[{"ref_block_num":17883,"ref_block_prefix":3817849555,"expiration":"2017-11-15T11:48:57","operations":[["feed_publish",{"publisher":"cyberfounder","exchange_rate":{"base":"0.600 GBGOS","quote":"1.000 GOLOS"}}]],"extensions":[],"signatures":["205686f61c1c7bb9604bd495c0ea872916c19433bfef2489ba898861a41a23ab2c4e19ca4cba6c8288002b8005e4b344ac8bf0ea7223289e811e90ddb2ae3f9a6c"]}]},"id":"000045dcf5718ad98650f8b4039adffba029f9fb"}
    th_a  database.cpp:2765 _apply_block
1954656ms th_a       database.cpp:2860             _apply_block         ] next_block.block_num(): 17884 
1954695ms th_a       database.cpp:807              _push_block          ] Failed to push new block:
10 basic_exception: tion
next_block.transaction_merkle_root == merkle_root: Merkle check failed
```

**Comment by Nemo1369** [comment]
@bitfag A known issue connected to #232. In process now.

**Comment by Nemo1369** [comment]
Fixed in #232
